Kerdy/Kerdy/Filter/Views/DateFilterView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  DateFilterView.swift
+//  Kerdy
+//
+//  Created by 이동현 on 11/3/23.
+//
+
+import UIKit
+import Core
+
+final class DateFilterView: UIView {
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "기간"
+        label.font = .nanumSquare(to: .bold, size: 15)
+        return label
+    }()
+
+    private lazy var dateStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.spacing = 10
+        stackView.distribution = .equalSpacing
+        return stackView
+    }()
+
+    lazy var startDateBtnView = DateSelectBtnView()
+    lazy var endDateBtnView = DateSelectBtnView()
+
+    lazy var waveLabel: UILabel = {
+        let label = UILabel()
+        label.font = .nanumSquare(to: .regular, size: 15)
+        label.text = "~"
+        return label
+    }()
+
+    init() {
+        super.init(frame: .zero)
+        setLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setLayout() {
+        addSubviews(titleLabel)
+        addSubviews(dateStackView)
```

Comment:
extension으로 addSubview를 한번에 할 수 있도록 구현해두었습니다!
```suggestion
        addSubviews(titleLabel, dateStackView)
```